docs/tutorial/creating_test.rst

```diff
@@ -0,0 +1,96 @@
+Creating a Test Dataset
+=======================
+
+This short tutorial makes use of a resqpy test function to generate a small dataset which can be used in other tutorials. Creating a RESQML dataset from scratch is significantly more complicated than reading an existing dataset and will be covered in later tutorials.
+
+Importing from a test module
+----------------------------
+For the following import to work, the resqpy code will need to have high priority in the environmental PATH. This can be achieved with something like:
+
+.. code-block:: python
+
+    import sys
+    sys.path.insert(0, '/hpcdata/beaiwd/git/resqml')
```

Comment:
FYI, I think with Sphinx the simpler syntax for python blocks is just using double colons `::`

```suggestion
For the following import to work, the resqpy code will need to have high priority in the environmental PATH. This can be achieved with something like::

    import sys
    sys.path.insert(0, '/hpcdata/beaiwd/git/resqml')
```

Reply:
It is yes - but it didn't work so well when I had multiline code blocks, and I wasn't sure about if it did python formatting either?